src/_shared/scss/_adverts-books.scss

```diff
@@ -73,6 +73,44 @@
         }
     }
 
+    &.advert--prominent-true > .advert__image-container {
+        flex-basis: calc(66.67% - #{$gs-gutter / 2});
+    }
+
+    &.has-no-flex {
```

Comment:
Small mistake here. the `has-no-flex` class will be added to the `html` element so you must do `.has-no-flex &` instead. Think of the `&` as a placeholder that will unfold (through bubble up) as the compounded current selector, so:

```
.selector1 {
  /* here & means .selector1 */
  .selector2 {
    /* here & means .selector1 .selector2 */
    .has-no-flex & {
      /* the selector above will thus end up as .has-no-flex .selector1 .selector2 */
    }

    &.has-no-flex {
      /* the selector above will end up as .selector1 .selector2.has-no-flex */
    }
}
```